circle.yml

```diff
@@ -16,7 +14,6 @@ dependencies:
     - cd libgit2/ && mkdir build && cd build && cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local && sudo cmake --build . --target install
     - go get -t -d -v ./...
     - go build -v
-    - go install -v
```

Comment:
realized that go build puts the compiled package in the current directory - so we don't need go install